### DISCUSSION TITLE
How to provide a cert for an ip like https://192.168.1.11:3000, in addition to https://localhost:3000

### DISCUSSION THREAD

**Discussion by Aarbel** [liuweiGL/vite-plugin-mkcert#19] [Q&A] [answered]
@liuweiGL thanks a lot for this amazing plugin

Cf my issue with Capacitor / Auth0 https://github.com/ionic-team/capacitor/issues/5149

Giving maybe more examples in your doc could be useful:

In which case and how should i use `source` `mkcertPath` `hosts` ?

**Comment by liuweiGL** [comment, marked as answer]
You can use  [hosts](https://github.com/liuweiGL/vite-plugin-mkcert#hosts) parameter:
```ts
export default defineConfig({
  server: {
    https: true
  },
  plugins: [
    mkcert({
           hosts: ['localhost','192.168.1.11']  
      })
]
})
```

**Reply by Aarbel** [reply]
Thanks a lot @liuweiGL, does putting it on `0.0.0.0` will generate a valid certificate for all domains like `192.168.1.11` ?

**Reply by liuweiGL** [reply]
No, it won't work.

**Comment by liuweiGL** [comment]
The plugin will generate a certificate for all local ips by default, i think it as same as `0.0.0.0`.

Ref: [getDefaultHosts](https://github.com/liuweiGL/vite-plugin-mkcert/blob/f03dedc0d8bc35c852e8797078ddb3b2c924850e/packages/plugin/src/lib/util.ts#L83)

**Reply by Aarbel** [reply]
Thanks @liuweiGL, i guess you should put this link in your readme doc ;)
